docs/manage/task-config-policies.rst

```diff
@@ -0,0 +1,165 @@
+to govern user behavior more closely at the workspace and cluster level.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+When implementing Config Policies, administrators should be aware of the following:
+
+#. Separate Priority Limits: Administrators can now set different priority limits for experiments
+   and NTSC (notebooks, tensorboards, shells, and commands) tasks. This allows for more granular
+   control over task prioritization and resource allocation.
+
+#. Priority Overrides: If a given scope has an invariant config policy set for a given task, and
+   that invariant configuration policy specifies ``priority``, this priority can still be overridden
+   using the ``SetJobPriority`` API endpoint. However, the new priority must not violate any
+   constraints defined in the applicable policies for that scope.
+
+These features provide administrators with greater flexibility in managing task priorities while
+still maintaining policy-based control over resource usage.
+
++------------------------------------------------------------------+
+| Visit the Config Policies Reference                              |
++==================================================================+
+| :ref:`task-config-policies-reference`                            |
++------------------------------------------------------------------+
+
+**************
+ Key Features
+**************
+
+-  Set policies via WebUI or CLI
+-  Define limits for resource usage, environment settings, and more
+-  Apply policies at different levels (cluster-wide or workspace)
+-  Override capabilities for specific user roles or groups
+-  Set different priority limits for experiments and NTSC (notebooks, TensorBoards, shells, and
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies using either the WebUI or the CLI.
+
+WebUI
+=====
+
+#. Sign in to the Determined WebUI as a cluster administrator or workspace administrator.
+#. Navigate to **Config Policies**.
+#. Choose **Experiments** or **Tasks** to display the editable configuration file.
+#. Define the policies then click **Apply**.
+#. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
+
+   # Update an existing policy
+   det policy update <policy_name> --config <updated_policy_config_file.yaml>
+
+   # Delete a policy
+   det policy delete <policy_name>
+
+*****************
+ Policy Examples
+*****************
+
+Here are some example policies:
+
+#. Limit resources.
+
+In the following example for an Agent resource manager (RM), the ``priority_limit`` is set to ``50``
```

Comment:
Let's use a different value for `priority_limit`. I chose 50 to illustrate a corner case I wanted to highlight. But I think we should use a smaller value (30 or less).